src/refract/RenderJSONVisitor.cc

```diff
@@ -223,6 +225,61 @@ namespace refract
         assign(renderer.get());
     }
 
+    namespace {
```

Comment:
I am asking you to move the anonymous namespace to be just below the refract namespace declaration. Not remove it completely.

```
namespace refract
{
    namespace {
```